src/neural-network.js

```diff
@@ -366,6 +392,22 @@ export default class NeuralNetwork {
     return true;
   }
 
+
+  /**
+   *
+   * @param options
+   * @private
+   */
+  _validateTrainingOptions(options) {
```

Comment:
This can be `static`, don't think it matters, but it can be.

Reply:
Good idea, there really is no need to have it on the object instead of on the class.